felix/cmd/calico-bpf/commands/conntrack.go

```diff
@@ -53,12 +55,15 @@ var conntrackCmd = &cobra.Command{
 
 type conntrackDumpCmd struct {
 	*cobra.Command
+	Version string `docopt:"<version>"`
+
+	version int
 }
 
 func newConntrackDumpCmd() *cobra.Command {
 	cmd := &conntrackDumpCmd{
 		Command: &cobra.Command{
-			Use:   "dump",
+			Use:   "dump [<version>]",
```

Comment:
I wonder if it should be a bit more verbose option

```suggestion
			Use:   "dump [--version=<version>]",
```